doc/protocol.md

```diff
@@ -88,7 +88,7 @@ like so:
 {
   "type":     "init",
   "msg_id":   1,
-  "node_id":  "n3",
+  "node_id":  "n1",
```

Comment:
Why change this?

Reply:
The description below says "The node_id field indicates the ID of the node which is receiving this message: here, the node ID is "n1"".  Thought the one in the message is a typo?

Reply:
Oh, yes, of course! That'll teach me to review patches first thing in the morning. ;-)